packages/x-components/static-docs/build-search-ui/web-archetype-development-guide.md

```diff
@@ -1,38 +1,33 @@
 ---
-title: Interface X Archetype Development
+title: Develop using Interface X Archetype
 tags:
   - development
   - archetype
-  - X Components archetype development
+  - X archetype development
   - x development
   - interface x
   - x components
   - archetype development
-  - archetype integration
 ---
 
-# Interface X Archetype Development
+# Develop using Interface X Archetype 
 
-In this tutorial, you’ll learn how to start developing with the Interface&nbsp;X&nbsp;Archetype
-project in your store in a matter of minutes, so you can create a search interface layer based on
-predefined features and components.
+In this tutorial, you’ll learn how to build a search and discovery layer based on predefined features and components in a matter of minutes using the Interface&nbsp;X&nbsp;Archetype.
 
 For this tutorial, the Empathy Search API is used, but you can use any search API. This guide
 requires knowledge of JavaScript and Vue.js.
 
 ::: note Before you begin
 
-To integrate Interface&nbsp;X&nbsp;Archetype as a search UI layer, you need:
-
-<br/>
+To use Interface&nbsp;X&nbsp;Archetype as a search UI layer, you need:
 
 - **Empathy Search API** (or any search API that you use to retrieve search data).
 - **Empathy Search Adapter** to communicate with the Empathy Search API (or any search adapter to
   connect with the search API you are using).
 
 :::
 
-##### Steps to start developing in a X Archetype project:
+##### Steps to build a search layer using the X Archetype project:
 
 1. **Clone** the X&nbsp;Archetype project and **initialize** your repository.
 2. Install the **project dependencies** and execute the project.
```

Comment:
Does **run** sound better than **execute**? I would also clarify that it is the development server the thing you are running.
```suggestion
2. Install the **project dependencies** and run the project development server.
```